web/src/pages/Vulnerability/AddVul.tsx

```diff
@@ -136,21 +141,36 @@ export default function AddVulnerability() {
     };
 
     const handleSubmit = async () => {
-        console.log(vuln);
+        if (fileListe) {
+            const data = new FormData();
+            files.forEach((file, i) => {
+                data.append(`file-${i}`, file, file.name);
+            });
+
+            axios
+                .post('https://httpbin.org/post', data)
```

Comment:
im not sure this is a good idea. since pentesters are reporting _vulnerabilities_, maybe it's best not to go through a 3rd-party service, just in case ? 

i think this is fine for the demo, but we might need to change that next sprint

Reply:
Ok no problem, we will see next sprint